e2e/src/test/java/org/jdbi/v3/e2e/testcontainer/mysql/TestMySQL.java

```diff
@@ -110,22 +112,22 @@ public String etag() {
     @Test
     public void testIssue2535PassesSingleLine() {
         String sqlScript = "CREATE PROCEDURE QWE() "
-            + "BEGIN "
-            + "END; "
-            + "DROP PROCEDURE IF EXISTS QWE;";
+                + "BEGIN "
```

Comment:
text block?

Reply:
this is actually a single line (the bug fix tests that as a single line vs. a multiline). Making it a text block adds newlines.